CHANGELOG.md

```diff
@@ -3,11 +3,29 @@
 ### Added
 
 - Add readiness probe to etcd pod. The readiness state will be reflected on `status.members.ready/unready`.
+- Tls support in backup/restore-operator.
+- Add spec validation in restore operator.
+- Add BackupStorageType and EtcdClusterRef.
```

Comment:
Add BackupStorageType to `EtcdRestore.Spec`. (Briefly what it does)
Add EtcdClusterRef ...